#
# Submitted-By: Marc Kleine-Budde, 2006-02-28
# Committed-By: Marc Kleine-Budde
#
# Error:
#
# xml2-config is not $SYSROOT aware
#
# Description: 
#
# this patch adds $SYSROOT support to xml2-config
#
# State:
# 
# not commited, but should be :)
#
---
# xml2-config.in |    2 +-
# 1 file changed, 1 insertion(+), 1 deletion(-)
#
Index: libxml2-2.6.32/xml2-config.in
===================================================================
--- libxml2-2.6.32.orig/xml2-config.in
+++ libxml2-2.6.32/xml2-config.in
@@ -1,6 +1,6 @@
 #! /bin/sh
 
-prefix=@prefix@
+prefix=${SYSROOT}@prefix@
 exec_prefix=@exec_prefix@
 includedir=@includedir@
 libdir=@libdir@
